libraries/pbrlib/genglsl/lib/mx39_microfacet.glsl

```diff
@@ -0,0 +1,30 @@
+#include "mx_microfacet.glsl"
```

Comment:
Always include the 1.38 code for all functions that were unchanged in 1.39.

Modus operandi:
Copy all files that are used by OpenPBR surface and have changes in 1.39. Prefix them with `mx39_`

Then, for each file:
Do a diff between 1.38 and 1.39. If a function, struct, or macro was modified in 1.39 prefix them with `mx39_`, `Mx39`, and `MX39_` (in that order). Find all call sites of the modified  function in the `mx39_` files, and update them. Remove any function, struct, or macro that was not changed since we will get it via the original files. This helps keep the backport small and reduces code duplication.

Extra step is to further prune code that is unused by OpenPBR Surface.